wdl-engine/src/eval/v1/task.rs

```diff
@@ -85,6 +102,99 @@ const OUTPUT_SCOPE_INDEX: ScopeIndex = ScopeIndex::new(1);
 /// visible.
 const TASK_SCOPE_INDEX: ScopeIndex = ScopeIndex::new(2);
 
+/// Gets the `container` requirement from a requirements map.
+pub(crate) fn container<'a>(
+    requirements: &HashMap<String, Value>,
+    default: Option<&'a str>,
+) -> Cow<'a, str> {
+    requirements
+        .get(TASK_REQUIREMENT_CONTAINER)
+        .or_else(|| requirements.get(TASK_REQUIREMENT_CONTAINER_ALIAS))
+        .map(|v| {
+            v.coerce(&PrimitiveType::String.into())
+                .expect("type should coerce")
+                .unwrap_string()
+                .as_ref()
+                .clone()
+                .into()
```

Comment:
This also needs to handle `Array[String]`

Reply:
Good eye! I'll get that fixed.

Reply:
It should also handle `"*"` appropriately.